src/nrsc5.c

```diff
@@ -175,6 +175,11 @@ static void nrsc5_init(nrsc5_t *st)
     pthread_create(&st->worker, NULL, worker_thread, st);
 }
 
+NRSC5_API void nrsc5_get_version(char **version)
```

Comment:
I would suggest using "const char **" to emphasize that the returned C string should not be modified nor freed.

Reply:
Done.